src/Test/Functional/Acceptance/BackupDbCest.php

```diff
@@ -0,0 +1,173 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MagentoCloud\Test\Functional\Acceptance;
+
+use CliTester;
+use Codeception\Example;
+use Exception;
+use Robo\Exception\TaskException;
+
+/**
+ * Checks database backup functionality
+ */
+class BackupDbCest extends AbstractCest
+{
+    /**
+     * {@inheritDoc}
+     * @param CliTester $I
+     */
+    public function _before(CliTester $I): void
+    {
+        // Do nothing
+    }
+
+    /**
+     * @param CliTester $I
+     * @param Example $data
+     * @throws Exception
+     * @dataProvider dataProviderMagentoCloudVersions
+     */
+    public function testBackUpDb(CliTester $I, Example $data): void
+    {
+        $expectedLogs = [
```

Comment:
Should this and $envMagento be brought into class properties instead of being passed as arguments?